### DISCUSSION TITLE
Remix page transitions

### DISCUSSION THREAD

**Discussion by monemagrafica** [remix-run/remix#5217] [Q&A]
is there a way to animate route transitions with exit animations? Anyone has an example repo? Thanx

**Comment by ttiimmaacc** [comment]
How come this questions keeps popping up and never gets addressed?

**Reply by sergiodxa** [reply]
It's the same thing as any React app, you can use things like Framer Motion, or even [View Transitions](https://remix.run/docs/en/main/components/link#unstable_viewtransition).

**Reply by ttiimmaacc** [reply]
I think that works only for basic routes, but what about for nested routes?
https://github.com/ttiimmaacc/hydro-page-animations

**Reply by HughEdwards1** [reply]
Is it actually the same?   
Trying something like <AnimatePresence> doesn't work because the content of the outlet is immediately replaced - trying to use useOutlet() causes issues because the loader data is undefined when using it. 

View Transitions are totally unsupported in firefox which isn't super useful
